File: src/main/java/Audio.java
import com.raylib.Raylib;

import static com.raylib.Raylib.*;

public class Audio {
    public static void PlayAudio(Sound audio)
    {
        PlaySound(audio);
        Parser.soundPlayed = true;
    }

    public static void LoadAudio()
    {
        Main.effect = LoadSound("resources/sounds/test.wav");
        Parser.soundLoaded = true;
    }
}

File: SpeechCS/src/engine/Audio.cs
using System.Numerics;
using Raylib_cs;
using static Raylib_cs.Raylib;
using static Raylib_cs.Color;

namespace Speech
{
    public class Audio
    {
        public static void PlayAudio(Sound audio)
        {
            PlaySound(audio);
            Parser.soundPlayed = true;
        }

        public static void LoadAudio(int i, string[] objec)
        {
            try
            {
                Program.effect = LoadSound("resources/sounds/" + objec[i] + ".wav");
                Parser.soundLoaded = true;
                Console.WriteLine("Loaded Audio: " + objec[i]);
            }
            catch (Exception e)
            {
                Console.WriteLine("Failed To Load Audio!");
            }
        }
    }
}
